provision/minikube/keycloak/config/kcb-infinispan-cache-remote-store-config.xml

```diff
@@ -242,6 +242,14 @@
             <expiration max-idle="3600000"/>
             <memory max-count="1000"/>
         </local-cache>
+        <local-cache name="crl" simple-cache="true">
+            <encoding>
+                <key media-type="application/x-java-object"/>
+                <value media-type="application/x-java-object"/>
+            </encoding>
+            <expiration lifespan="-1"/>
+            <memory max-count="1000"/>
```

Comment:
Consider removing this file in this PR, or a follow-up issue. We don't use it anymore.